### PR TITLE
Auto

### PR DESCRIPTION
**Pull request by Rylan-Meilutis**
Working auto

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 1]
```diff
@@ -56,8 +66,8 @@ public Climber(OperatorInterface oi) {
         climber.configPeakOutputReverse(-1, TIMEOUT);
         climber.setNeutralMode(NeutralMode.Brake);
         climber.configSupplyCurrentLimit(climberLimit, TIMEOUT);
-        climber.configForwardLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 0);
-        climber.configReverseLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 0);
+        // climber.configForwardLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 0);
```
Comment (JackDunfey) [inline src/main/java/frc/robot/subsystems/Climber.java]: Why were these removed?

Comment (mmarchetti) [reply src/main/java/frc/robot/subsystems/Climber.java]: The limit switches aren't compatible with the Falcon, so we are implementing the limit switch checks ourselves.

Comment (JackDunfey) [reply src/main/java/frc/robot/subsystems/Climber.java]: Oh ok
